packages/orbit-components/src/InputSelect/__tests__/index.test.tsx

```diff
@@ -216,6 +216,50 @@ describe("InputSelect", () => {
     expect(screen.getAllByRole("option")).toHaveLength(1 + 2 + 1 + 4); // (1 previously selected, 2 asian, 1 american, 4 all)
   });
 
+  it("when readOnly is true", async () => {
```

Comment:
`when` statements are usually associated with `describe`, and not `it`. To avoid more nesting, this can be simply

```
it("can be readOnly",
```